Makefile

```diff
@@ -20,6 +20,12 @@ BENCH_OUTPUT ?= stdout
 .PHONY: all
 all: modcheck swagger binary binary-minimal binary-debug cli bench exporter-minimal verify-config test covhtml check
 
+.PHONY: minimal
```

Comment:
Can we give this another name, for it to be easily distinguishable from minimal binaries build ?

Reply:
Maybe minimal-conformance-check, since we check if it is meeting criteria in " check test covhtml " .

Reply:
Renamed as suggested